salvador/ContentView.swift

```diff
@@ -9,55 +9,182 @@ import SwiftUI
 
 struct ContentView: View {
```

Comment:
You could think of giving the `ContentView` a proper name. (e.g. `GenerateImageView`)
If you would create a view model for it in the future a proper name would be `GenerateImageViewModel`

Reply:
When I change the contentView to GenerateImageView the compile feil , Its not obligatory to use ContentView as main roof  of the application?

Reply:
Thx I found out how to change the name of the view :)